source/installation-guide/kibana/index.rst

```diff
@@ -104,12 +110,51 @@ Kibana installation and configuration
 
     .. include:: ../../_templates/installations/elastic/common/enable_kibana.rst
 
-    The first time you use a browser to access Kibana, the browser will show a warning message stating that the certificate was not issued by a trusted authority. This can be accepted either by clicking on ``Advanced options`` to add an exception or, for increased security, by importing the ``ca.crt`` previously created to the Certificate Manager of each browser that will access the Kibana interface.
+    With the first access to Kibana, the browser shows a warning message stating that the certificate was not issued by a trusted authority. This can be accepted either by clicking on ``Advanced options`` to add an exception or, for increased security, by importing the ``ca.crt`` previously created to the Certificate Manager of each browser that will access the Kibana interface.
 
     .. note:: The Kibana service listens to the default port 5601. The browser address will be: ``https://<kibana_ip>:5601`` replacing <kibana_ip> by the Kibana server IP.
 
+Disabling repositories
+~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: ../../_templates/installations/elastic/common/disabling_repositories_explanation.rst
+
+
+.. tabs::
+
+
+  .. group-tab:: APT
+
+
+    .. include:: ../../_templates/installations/elastic/deb/disabling_repositories.rst
+
+
+
+  .. group-tab:: Yum
+
+
+    .. include:: ../../_templates/installations/elastic/yum/disabling_repositories.rst
+
+
+
+  .. group-tab:: ZYpp
+
+
+    .. include:: ../../_templates/installations/elastic/zypp/disabling_repositories.rst
+
 
 Next steps
-----------
+~~~~~~~~~~
+
+Once the Wazuh - Elastic Stack environment is ready, a Wazuh agent can be installed in every endpoint to be monitored. The Wazuh installation guide is available for most operating systems and it can be found :ref:`here<installation_agents>`.
+
+Uninstall Kibana
+~~~~~~~~~~~~~~~~
+
+To uninstall Kibana:
+
+    .. code-block:: console
+
+      # apt-get remove kibana
 
-Once the Wazuh - Elastic Stack environment is ready, you may install a Wazuh agent in every endpoint to be monitored. The Wazuh installation guide is available for most operating systems and it can be found :ref:`here<installation_agents>`.
+There are files marked as configuration and data files. Due to this designation, the package manager doesn't remove those files from the filesystem. The complete files removal action is a user responsibility. It can be done by removing the folder ``/var/lib/kibana`` and ``/etc/kibana``.
```

Comment:
To uninstall Kibana we should add tabs like the ones used in Elasticsearch and Wazuh sections to include the Yum and Zypp options.  Besides, when using `apt-get remove` with the flag `--purge` all the files marked as configuration and data files will be removed.

Reply:
I will investigate this.